packages/sw-background-sync-queue/src/lib/queue.js

```diff
@@ -47,6 +73,12 @@ class Queue {
 		}
 	}
 
+	/**
+	 * clean up the queue, deleting all the tasks who are either damaged or
+	 * whose maxAge has expired
```

Comment:
Could you expand on what results in a task that is damaged? It may be worth documenting what we mean here. I think maxAge expiration is pretty clear otherwise.